### DISCUSSION TITLE
Announcement: GitHub Issues Reset

### DISCUSSION THREAD

**Discussion by sargonas** [Chia-Network/chia-blockchain#6920] [Announcements]
**Tl;dr - There is an unmanageable amount of GitHub issues that are either old, not issues, or lost in the noise. We will be doing a “reset” and starting from scratch in the coming days.**

We’re working on a better way to help and support the community!

Due to an overwhelming amount of GitHub Issues—many of which are not actual bugs and a majority of which are from before the current version of Chia was released—in the coming days we will be conducting a GitHub Issue Reset.

There are an overwhelming number of “issues” that are little more than duplicates of “I can’t sync” or redundant posts about netspace growth rate, pooling, etc., with many more that are asking for general support (which should be posted elsewhere such as [Discussions](https://github.com/Chia-Network/chia-blockchain/discussions) or [Keybase](https://keybase.io/team/chia_network.public)). Most were opened for the above reasons and then subsequently abandoned by their poster when they resolved their issue on their own or new releases were launched, making them out of date.

GitHub Issues are meant to be a place for the Developers to work with the community to resolve bugs and issues, communicate with the open source community over the Pull Requests and feature enhancements they provide. Now that Chia has dedicated resources available for this task, our goal is to stay on top of them regularly so we can solve your issues, but to do that, we need to start from a manageable baseline.

We will be conducting a reset later this week that will mass-close most issues, save for a handful being kept open for one reason or another. The closure will be accompanied with notes that direct submitters on the best course of action to follow up if they still need assistance, with guidance on how to find support in [Discussions](https://github.com/Chia-Network/chia-blockchain/discussions) or [Keybase](https://keybase.io/team/chia_network.public), documentation in the Wiki, and how to re-open the issue for manual review by Chia staff if they think it's an Issue that warrants further attention.

Additionally, we will begin employing the same automated GitHub Job Actions used by many other organizations currently, and auto-respond with a warning to stale issues that have had no updates beyond a week, and automatically closing them a few days after if they are not commented on, in further attempts to keep things manageable and follow best practices.

Do know, however, that none of these actions are permanent or irreversible. We will always endeavor to surface your issues to the right people. Our concern is not about obscuring the _quantity_ of issues, but about ensuring the _quality_ of what is there, and managing the ability to filter through them to ensure the issues that _need_ addressing by the Chia Devs are seen.

**Comment by joao-de-melo** [comment]
I used to think this project had a good foundation relying on good development practices. Not anymore definitely. Shameful to delete/ignore potentially useful data provided by users, even if it was only 1% useful. Just do your homework and tidy up the backlog. What a joke.

**Reply by ByronAP** [reply]
come on man it's 99.9% junk and frustrating for people who actually know how to file an issue (and someone should get smacked for posting issues like those that are just titles in the pool repo... lol)

**Reply by sargonas** [reply]
Nothing is being deleted, it's being _closed_. The issues are still there, and anyone can still look at and read them with two extra clicks to enable a filter to show them. They are simply being closed and filtered out of the initial top-level view for being stale/out of date/no longer relevant.. and if any of them ARE any of those things, all the OP has to do is ask to have it reopened when they get the email notification, and we'll do just that as well as try to resolve it for them.

**Reply by joao-de-melo** [reply]
So many years creating/perfecting development practices to enhance/maximize the impact of feedback loop in software development. I am shocked regardless, not something a serious project would do.

**Reply by JayDoubleu** [reply]
Simple solution... Close with message asking to reopen if issue truly still exists and just work through those.

Win-Win

**Reply by sargonas** [reply]
> Simple solution... Close with message asking to reopen if issue truly still exists and just work through those.
> 
> Win-Win

That's literally what I'll be doing, with just a bit more in-depth instructions on the comment to guide people to the right places for support for non "issues" and we will re open the ones they ask us to that are valid. :)

**Reply by dorkmo** [reply]
@lyncodev i had to turn off notifications on the chia repository. I dont know how anyone could keep notifications on without going crazy

**Reply by sargonas** [reply]
> i had to turn off notifications on the chia repository. I dont know how anyone could keep notifications on without going crazy

Can confirm. Am crazy.

**Comment by ghost** [comment]


**Reply by gromran** [reply]
90% of issues are from dau's, who consider their layer 8 problems to be bugs

**Reply by wallentx** [reply]
Sometimes you need to just need to be able to work from a clean slate, as opposed to working against 1.4k issues. Issues that are still relevant and important enough can be reopened. It makes more sense to do exactly what they are doing. I see this interface of the project as community contribution, rather than a customer support ticket system.

**Reply by indyramp** [reply]
If you read the actual words instead of assuming the worst, it makes a lot more sense. 

If you're volunteering to work a week for free, just say that @gumba4

**Comment by athkalia** [comment]
Can't you use something like this to close all the stale issues instead:

https://github.com/probot/stale

In a few weeks only the issues that are being discussed will stay open.

**Reply by sargonas** [reply]
Funny you mention that, I plan to incorporate something just like this going forward once we have it all under control. I have been going back and fourth on just turning it on at the start and letting *it* do the pruning of all the old issues as well, and I might still do that, but for now am a little uncertain how it might behave with over 1k issues, vs a rolling window of 100 a week or so. Definitely going to implement something like this going forward though, that part is for certain.

**Reply by athkalia** [reply]
This is pretty mainstream these days in large projects. I guess you can give it a try, you have nothing to lose - worst-case scenario it hits the GitHub API rate limit or something, although unlikely given that it's in the thousand requests per hour: https://docs.github.com/en/developers/apps/building-github-apps/rate-limits-for-github-apps

**Comment by wolfrage76** [comment]
![gene hoffman](https://user-images.githubusercontent.com/75458290/123439964-48ee6a00-d5a0-11eb-947c-19c288d61801.jpg)

**Comment by ddowhy** [comment]


**Reply by ByronAP** [reply]
> been doing professional software engineering for 20+ years at big big name software companies (20k-130k+ employees) and I have never done, come across, or even 'heard of' a team doing a 'issue reset'. It seems to me you need to hire some permanent change management to review and moderate these issues instead of copping out by doing a 'reset'....there's probably a lot of legitimate issues in there that may come back to bite you later if you just trash them all.

READ!!! It is not trashing everything...

**Reply by cjedwards** [reply]
> > been doing professional software engineering for 20+ years at big big name software companies (20k-130k+ employees) and I have never done, come across, or even 'heard of' a team doing a 'issue reset'. It seems to me you need to hire some permanent change management to review and moderate these issues instead of copping out by doing a 'reset'....there's probably a lot of legitimate issues in there that may come back to bite you later if you just trash them all.
> 
> READ!!! It is not trashing everything...

^^^

In fact not trashing ANYTHING!
I love all the fud posters.. end of the world!!! runnnn

**Comment by Tj-Swanky** [comment]
📥 HELLO INVESTOR 👋
        
✍️  STOCKTRADE-INDEX Investment company is very expectional💯

📊"𝐈𝐟 𝐲𝐨𝐮 𝐧𝐞𝐯𝐞𝐫 𝐭𝐫𝐲,
𝐘𝐨𝐮 𝐰𝐢𝐥𝐥 𝐧𝐞𝐯𝐞𝐫 𝐤𝐧𝐨𝐰..
📊𝐃𝐨𝐧'𝐭 𝐝𝐨𝐮𝐛𝐭 𝐲𝐨𝐮𝐫 𝐚𝐛𝐢𝐥𝐢𝐭𝐲 𝐭𝐨 𝐬𝐮𝐜𝐜𝐞𝐞𝐝..
📊𝐘𝐨𝐮 𝐦𝐚𝐲 𝐛𝐞 𝐣𝐮𝐬𝐭 𝐚 𝐬𝐭𝐞𝐩 𝐚𝐰𝐚𝐲 𝐭𝐨 𝐬𝐮𝐜𝐜𝐞𝐬𝐬..
📊𝐒𝐭𝐚𝐲 𝐟𝐨𝐜𝐮𝐬 ✅
📊𝐒𝐭𝐚𝐲 𝐩𝐨𝐬𝐢𝐭𝐢𝐯𝐞 ✅
📊𝐈𝐧𝐯𝐞𝐬𝐭𝐦𝐞𝐧𝐭 𝐢𝐬 𝐭𝐡𝐞 𝐤𝐞𝐲 𝐭𝐨 𝐬𝐮𝐜𝐜𝐞𝐬𝐬 ✅ Minimum investment plan1 30$ to earn 18% profit after 12 hours

📊𝐃𝐌 𝐍𝐎𝐖👨‍💻👇
          

http://stocktrade-index.com/?ref=signup
